tests/sys/psa_crypto_cbor_encoder/test_psa_crypto_enc_dec_keypair.c

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2023 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Tests CBOR encoding of a PSA Crypto asymmetric key pair slot.
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include "embUnit.h"
+#include "psa/crypto.h"
+#include "psa_crypto_cbor_encoder.h"
+#include "tests_psa_cbor_enc_dec.h"
+
+static uint8_t cbor_encoded_data[] = {
```

Comment:
Please document how this binary data was obtained and what it represents. Adding a comment with CBOR diagnostic notation gives extra points :)

The same applies to the other files.

Reply:
I added some documentation, is that what you meant?

Reply:
Hum, it actually doesn't tell much to be honest. More important is the documentation of the format in `psa_crypto_cbor_encoder.h` which is nice and concise, maybe just refer to that here (and in the other files).